app/src/main/java/com/nilhcem/blenamebadge/ui/badge_preview/Cell.kt

```diff
@@ -2,6 +2,4 @@ package com.nilhcem.blenamebadge.ui.badge_preview
 
 import android.graphics.Rect
 
-internal class Cell {
-    var list = mutableListOf<Rect>()
-}
+data class Cell(var list: MutableList<Rect> = mutableListOf())
```

Comment:
```suggestion
data class Cell(val list: List<Rect> = mutableListOf())
```

Reply:
need to add things to the list, cant use list

Reply:
You can

Reply:
And please don't resolve conversations, I'm going to have to stop reviewing your PRs if you resolve conversations manually. It is extremely annoying

Leave them open if you are not going to do it

Reply:
@iamareebjamal I did not resolve it sir, it got resolved automatically when I clicked the commit suggestion button

Reply:
Sir, It says 'A type annotation is required on a value parameter'